### ISSUE TITLE
Enable magnification in this sim

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/faradays-law#173]
As part of the next publication of this sim, we would like to include magnification. This issue is to enable it in this sim and make sure all is working well with the feature. Assigning to myself to set the `supportsZoom` flag in package.json, and some review.

Also assigning to @jbphet so he is aware since he is doing work in this repo related to sound and will likely see through publication of this sim with sound, magnification, and alternative input. Feel free to just un-assign or let me know if you have any questions.

**Comment by jessegreenberg** [comment]
Magnification is now enabled in faradays-law by default.

**Comment by jessegreenberg** [comment]
Everything seems to be working well except the Checkboxes. It looks like listeners on them aren't interrupted if a panning operation  happens on them. Pressing down on a checkbox and then dragging will eventually click the checkbox when the pointer is released. I will look into that.

**Comment by jessegreenberg** [comment]
Another issue I see is that the freely movable magnet does not pan the view when it leaves the zoomed in window. This sort of panning does happen with mouse dragging so it should happen with keyboard dragging as well.

**Comment by jessegreenberg** [comment]
> Another issue I see is that the freely movable magnet does not pan the view when it leaves the zoomed in window. This sort of panning does happen with mouse dragging so it should happen with keyboard dragging as well.

This should be done in the above commit. After https://github.com/phetsims/scenery/issues/1072 is addressed this will be ready for review.

**Comment by jessegreenberg** [comment]
https://github.com/phetsims/scenery/issues/1072 has been resolved. Zoom feature appears to be working as I would expect for everything now. @emily-phet reassigning to you to review (or assign someone else to), as I recall that was done in ratio-and-proportion.

**Comment by emily-phet** [comment]
@jbphet Can you take a look at this?

**Comment by jbphet** [comment]
The behavior seems good (I just tested it on master using my Dell XPS Windows device that has a touchscreen), and I looked through the commits, and I don't see anything obviously amiss.  I don't know enough about how this feature works to give the commits a very informed review though.  @jessegreenberg - is that enough to call this good, or should someone else look at it.

**Comment by emily-phet** [comment]
I'll check that QA has a good list of things to review for pan/zoom behavior. I think that will be sufficient.
